src/System.Diagnostics.Debug/src/System.Diagnostics.Debug.csproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+    <Configuration Condition="'$(Configuration)'==''">$(TargetOS)_Debug</Configuration>
```

Comment:
TargetOS isn't set by the time this is evaluated so you are essentially setting it to `_Debug` which isn't going to help.